Update documentation for minimum_epoch_interval

jaronsummers opened the pull request:
google.protobuf.Duration's JSON behavior is not super intuitive so updating the example to be reflective of real usage will hopefully save someone some time

jaronsummers commented on pkg/proto/configuration/blobstore/blobstore.proto:
```diff
@@ -576,12 +576,14 @@ message LocalBlobAccessConfiguration {
     // bookkeeping consumes up to 12*24*365*16 B = ~1.68 MB of space if
     // the system were to operate for a full year without blocks being
-    // released. Setting this to 1s would blow this up by a factor 300.
+    // released. Setting this to "1s" blows this up by a factor 300.
```
I rephrased here just to keep the line length short.

EdSchouten commented:
Ah, great. Thanks for addressing this. Older versions of the Protobuf library would allow you to fill in any duration string supported by Go's `time.Duration`. So `5m`, `10h`, etc. was all valid. Later on they restricted it to just accept seconds, for consistency with the Protobuf implementations of other languages. This documentation hasn't been updated to reflect that.

EdSchouten commented on pkg/proto/configuration/blobstore/blobstore.proto:
```diff
@@ -576,12 +576,14 @@ message LocalBlobAccessConfiguration {
     //
-    // Recommended value: 5m
+    // Recommended value: "300s"
+    // In JSON google.protobut.Duration is a string, time in seconds,
```
My recommendation would remove this sentence, though, as we have many google.protobuf.Duration fields in our configuration and this isn't the most logical place to document it.

Maybe we should just link to https://developers.google.com/protocol-buffers/docs/proto3#json from README.md?

jaronsummers replied:
That makes sense to me. Apologies for not checking the rest of the codebase for similar things.

jaronsummers commented:
I didn't see any really obvious places to stick a link in the readme, but I removed the suggested sentence, and also updated my quotes to be single-quotes like the other examples in that file.
